server/requirements.txt

```diff
@@ -15,4 +15,5 @@ timm==0.6.13
 typeguard==2.13.3
 accelerate==0.18.0
 pytesseract==0.3.10
-gradio==3.24.1
\ No newline at end of file
+gradio==4.22.0
+pillow>=10.3.0 # not directly required, pinned by Snyk to avoid a vulnerability
```

Comment:
**Suggestion**: After pinning "pillow" to version 10.3.0 to address the vulnerability, it's important to verify that this version does not conflict with other dependencies or project functionalities. Consider running a compatibility check and a full regression test suite. <br> **Code Suggestion**: 
 ```
 +pillow>=10.3.0 # Verify compatibility and perform thorough testing.
 ```